Wikipedia/Code/TalkPageViewController.swift

```diff
@@ -453,7 +459,17 @@ class TalkPageViewController: ViewController {
         if UIAccessibility.isVoiceOverRunning {
             UIAccessibility.post(notification: UIAccessibility.Notification.announcement, argument: title)
         } else {
-            WMFAlertManager.sharedInstance.showBottomAlertWithMessage(title, subtitle: subtitle, image: image, dismissPreviousAlerts: true)
+            WMFAlertManager.sharedInstance.showBottomAlertWithMessage(title, subtitle: subtitle, image: image, type: .normal, customTypeName: nil, dismissPreviousAlerts: true)
+        }
+    }
+
+    fileprivate func subscriptionErrorAlert() {
+        let title = TalkPageLocalizedStrings.subscriptionFailed
+
+        if UIAccessibility.isVoiceOverRunning {
+            UIAccessibility.post(notification: UIAccessibility.Notification.announcement, argument: title)
+        } else {
+            WMFAlertManager.sharedInstance.showBottomAlertWithMessage(title, subtitle: nil, image: UIImage(systemName: "exclamationmark.circle"), type: .custom, customTypeName: "subscription", dismissPreviousAlerts: true)
```

Comment:
I'm unable to see the error message in the darker themes on this one:

![IMG_0846](https://user-images.githubusercontent.com/3620196/197632996-07edc6a2-2a53-4d4f-a6cb-5d7f90221218.PNG)

Reply:
Updated!